.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```

Comment:
I'd love to be assigned to feature requests to keep track

Reply:
I think the label is "enhancement" in this repo and sure we can add that